P/Pythia8/build_tarballs.jl

```diff
@@ -0,0 +1,42 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Pythia8"
```

Comment:
It looks like the name of the library is "PYTHIA", not "Pythia8", we try to keep the same name as the upstream project, including capitalisation.

Reply:
I'm happy to change, but Pythia 6 is all Fortran (hence the caps), while Pythia8 is all C++. I think maybe somebody would like both. Do you have a suggestion?

Reply:
We can have multiple version of the same package at the same time, see e.g. [OpenBLAS](https://github.com/JuliaPackaging/Yggdrasil/tree/a43603d4e548800b57ff3f4513eabdc10a0dc4f2/O/OpenBLAS) or [LLVM](https://github.com/JuliaPackaging/Yggdrasil/tree/a43603d4e548800b57ff3f4513eabdc10a0dc4f2/L/LLVM).

Reply:
OK. Should I close this and put it into the right dir structure?

Reply:
Why need to close?

Reply:
Ah. Right. I have a terminal and don't need to go through the web interface :-)